Project/Assets/Prairie/Framework/Script/Interaction/Prompt.cs

```diff
@@ -1,6 +1,7 @@
 using UnityEngine;
 using System.Collections;
 
+[AddComponentMenu("Prairie/Interactions/Prompt")]
```

Comment:
`Prompt` is automatically added in all cases where they would be useful. So I think we introduce confusion as including alongside the collection of interactions that can be added to an object. It's certainly related to interactions, but it in itself is not an interaction.

Maybe inside _Prairie/Support_ or _Prairie/Utility_?